scripts/codemod/helpers/codemodHelpers.js

```diff
@@ -0,0 +1,284 @@
+// @flow strict
+
+/**
+ * IMPORTANT NOTE
+ * Flow is used in these codemods with the only intention of providing better documentation on hover for each helper function while developing codemods and to increase code readability. Due to the complexity of Flow typing, we use generic types to prevent Flow form complaining. Simple props such as strings or booleans are typed to facilitate the usage of each helper function.
+ */
+
+// $FlowFixMe[unclear-type]
+type GenericType = any;
+
+type InitializeType = {
+  api: { jscodeshift: GenericType },
+  file: { source: GenericType },
+};
+
+/**
+ * initialize: Sets the boilerplate required to work with jscodeshift
+ */
+const initialize = ({ api, file }: InitializeType): $ReadOnlyArray<mixed> => {
+  const j = api.jscodeshift;
+  const src = j(file.source);
+  return [j, src];
+};
+
+type IsNotGestaltImportType = {
+  importDeclaration: GenericType,
+};
+
+/**
+ * isNotGestaltImport: Validates a Gestalt import path returning true if it matches 'gestalt'.
+ * E.g. import { Box } from 'gestalt' // true
+ * E.g. import { Box } from 'gestaltExtensions/box' // false
+ */
+const isNotGestaltImport = ({ importDeclaration }: IsNotGestaltImportType): boolean =>
```

Comment:
This seems like it would be more straightforward as `isGestaltImport`

Reply:
DONE